packages/expo-av/src/index.ts

```diff
@@ -1,5 +1,6 @@
 import * as Audio from './Audio';
 export { Audio };
+export { AudioMode, InterruptionModeAndroid, InterruptionModeIOS } from './Audio.types';
```

Comment:
Wouldn't something like this enable more flexibility and less boilerplate?
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#export--as-ns-syntax
```suggestion
export * from './Audio.types';
```